opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -26,3 +26,4 @@
     Sequence[float],
 ]
 Attributes = Optional[Dict[str, AttributeValue]]
+AttributesFormatter = Callable[[], Optional[Dict[str, AttributeValue]]]
```

Comment:
```suggestion
AttributesFormatter = Callable[[], Attributes]
```

Or am I missing something about these types?